judgels-backends/judgels-server-app/src/main/java/judgels/sandalphon/LocalGit.java

```diff
@@ -88,12 +94,20 @@ public void addAll(Path rootDirPath) {
     }
 
     @Override
-    public void commit(Path rootDirPath, String committerName, String committerEmail, String title, String description) {
+    public void commit(
```

Comment:
Can we revert all formatting changes in this file that are not related to the PR? It is a bit confusing 😄